### PR TITLE
Fix ormtests project

### PR DESCRIPTION
**Pull request by nbeteram**
The tests have been fixed.

### CODE REVIEW

**Review by Mouwrice — APPROVED** [review]
Jippie!

File: backend/orm_functions/project.ts [hunk 1]
```diff
@@ -374,77 +371,61 @@ export async function filterProjects(
...
 
-                const fullyAssignedX = x.positions === sum_x ? 1 : 0;
-                const fullyAssignedY = y.positions === sum_y ? 1 : 0;
+            const fullyAssignedX = x.positions === sum_x ? 1 : 0;
+            const fullyAssignedY = y.positions === sum_y ? 1 : 0;
 
-                return fullyAssignedX - fullyAssignedY;
-            }
+            console.log(fullyAssignedX);
+            console.log(fullyAssignedY);
```
Comment (Mouwrice) [inline backend/orm_functions/project.ts:421]: console.log(), begone

**Review by Mouwrice — CHANGES_REQUESTED** [review]

File: backend/orm_functions/project.ts (hunk 1 above)
Comment (KoenDesplenter) [reply backend/orm_functions/project.ts]: I removed the logs

**Review by KoenDesplenter — APPROVED** [review]
Well done

**Pull request merged**